### DISCUSSION TITLE
Installation failed

### DISCUSSION THREAD

**Discussion by wschiro** [fredlcore/BSB-LAN#689] [General]
I want to install the adapter with the url https://github.com/fredlcore/BSB-LAN
but it failed. The error means "corrupted files, try later.
the same, if I try to install the 4.1

**Comment by fredlcore** [comment]
Apparently, something went wrong. For starters, the fact that you neither provide any of the details required to further investigate, nor even use the proper way indicated in the manual.

**Reply by wschiro** [reply]
The iobroker-adapter ist not in the npm-repo and not in the GitHub-repo. thatswhy I tried it with the url.
the manual linked me to the installation of the BSB, but it doesn't helped. The hardware of the usb-lan I have installed und it worked. 
ERROR: Process exited with code 25
weniger

mehr
$ iobroker url https://github.com/fredlcore/BSB-LAN --host raspi37 --debug
install fredlcore/BSB-LAN#87989b33c67c0ce02128110c2442f856a60418f2
NPM version: 10.8.2
Installing fredlcore/BSB-LAN#87989b33c67c0ce02128110c2442f856a60418f2... (System call)
npm warn tarball tarball data for github:fredlcore/BSB-LAN#87989b33c67c0ce02128110c2442f856a60418f2 (null) seems to be corrupted. Trying again.
npm warn tarball tarball data for github:fredlcore/BSB-LAN#87989b33c67c0ce02128110c2442f856a60418f2 (null) seems to be corrupted. Trying again.
npm error code ENOENT
npm error syscall opennpm error path /home/iobroker/.npm/_cacache/tmp/git-cloneta1V8a/package.jsonnpm error errno -2npm error enoent Could not read package.json: Error: ENOENT: no such file or directory, open '/home/iobroker/.npm/_cacache/tmp/git-cloneta1V8a/package.json'
npm error enoent This is related to npm not being able to find a file.npm error enoentnpm error A complete log of this run can be found in: /home/iobroker/.npm/_logs/2024-12-14T09_54_47_424Z-debug-0.log
npm warn tarball tarball data for github:fredlcore/BSB-LAN#87989b33c67c0ce02128110c2442f856a60418f2 (null) seems to be corrupted. Trying again.npm warn tarball tarball data for github:fredlcore/BSB-LAN#87989b33c67c0ce02128110c2442f856a60418f2 (null) seems to be corrupted. Trying again.npm error code ENOENTnpm error syscall opennpm error path /home/iobroker/.npm/_cacache/tmp/git-cloneta1V8a/package.jsonnpm error errno -2npm error enoent Could not read package.json: Error: ENOENT: no such file or directory, open '/home/iobroker/.npm/_cacache/tmp/git-cloneta1V8a/package.json'npm error enoent This is related to npm not being able to find a file.npm error enoentnpm error A complete log of this run can be found in: /home/iobroker/.npm/_logs/2024-12-14T09_54_47_424Z-debug-0.log
host.raspi37 Cannot install fredlcore/BSB-LAN#87989b33c67c0ce02128110c2442f856a60418f2: 254
ERROR: Process exited with code 25

**Reply by fredlcore** [reply]
Ok, good, now we know that you are not talking about my BSB-LAN adapter (the hardware) but the IOBroker BSB-LAN adapter (software). Unfortunately, I don't use IOBroker and I'm also not the one who has written or maintains the adapter, so you would have to get in touch with the person in charge (which, if I'm not mistaken, is @hacki11). 
However, from what I can tell, you have to use the BSB-LAN iobroker adapter URL, not the one of my repository.

**Reply by hacki11** [reply]
Just have a look within iobroker, there you will find adapters. Search for BSB and you will directly be able to install.

Here is the actual adapter for iobroker https://github.com/hacki11/ioBroker.bsblan

**Reply by fredlcore** [reply]
Thanks a lot, @hacki11, for the quick feedback!
